blog/2020/08/juliacon-2020-wrapup.md

```diff
@@ -0,0 +1,38 @@
+@def title = "JuliaCon 2020 Wrap-up"
+@def authors = "The JuliaCon 2020 Organizing Committee"
+@def published = "10 August 2020"
+@def rss_pubdate = Date(2020, 8, 10)
+@def rss = """JuliaCon 2020 Wrap-up"""
+
+JuliaCon 2020, the 7th annual Julia conference, has come to a close. This year, given the global COVID-19 Pandemic, the [organizing committee](https://juliacon.org/2020/committee.html) made the decision to move the conference from Lisbon Portugal into a [completely virtual experience](https://discourse.julialang.org/t/juliacon-2020-is-going-virtual/38327). This virtual format presented many challenges that we had to move quickly to solve. Avik and Jane (co-Chairs of JuliaCon) detailed some of the thinking and rationale for the high level design decisions that were made for this year's JuliaCon in the [final episode of the inaugural JuliaCon podcast](https://www.youtube.com/watch?v=hbHmH0fezjs). 
```

Comment:
Should there be a comma between `Lisbon Portugal`?

Reply:
yes